test/index.js

```diff
@@ -86,19 +86,25 @@ test('find an item in the collection', () => {
 	const coll = new Collection();
 	coll.set('a', 1);
 	coll.set('b', 2);
-	assert.strictEqual(coll.find(x => x === 1), 1);
-	assert.strictEqual(coll.find(x => x === 10), undefined);
+	assert.strictEqual(
+		coll.find((x) => x === 1),
```

Comment:
```suggestion
		coll.find(x => x === 1),
```
same for other occourances

Reply:
@notsugden imo, the best thing to do here would be to setup a forced git hook to run prettier, as clearly some contributors' prettier setups are using different settings. (https://github.com/discordjs/collection/pull/31#pullrequestreview-569562123)

Reply:
Switched tests to depend on jest in https://github.com/discordjs/collection/pull/31/commits/ad21da68c7d3bcc2ae803c338333887d1ecbf719